lib/MusicBrainz/Server/Form/Work.pm

```diff
@@ -68,6 +68,35 @@ sub is_empty_attribute {
     return !$field->field('type_id')->value && !$field->field('value')->value;
 }
 
+sub validate_languages {
+    my $self = shift;
+
+    my @languages = $self->field('languages')->fields;
+
+    my $is_valid = 1;
+
+    # If we have only one language, then it can be whatever
+    # If we have two or more, we check it's not mul (248) nor zxx (486)
+    # because combining those with anything else doesn't make sense
```

Comment:
Previous comments have shown that combining other language(s) with `mul` can be used to specify that there are more languages which:
* either have to be determined ,
* or are not supported by MusicBrainz.

It would be helpful to know how many legitimate occurrences of such combination are currently used in the database.

Reply:
As I commented above, this seems to not be the ideal way to do this anyway, unless you specifically know there's at least two additional languages so that "multiple" is still true. But I can run a query.

Reply:
That assumes eng + mul means "english, plus other multiple languages" rather than just "multiple languages, including english," which is how I'd interpret it, but perhaps you're right that it's not an ideal way.

Reply:
I agree it is not ideal but it is still better than checking annotation to make sure “English” means just “English” and not “English plus possibly other languages”. Anyway it matters only if it is used.

Reply:
We have 30 of those (well, 29 because one was a clearly wrong mess I'm getting rid of). Some seem to wrongly have [multiple] and a few seem to be intended as "something I don't know" (example annotation: "English and an unknown second language."). If the intention is to have a way to indicate "I don't know" then we should have an "[unknown]" option, but I think this is so minor that it is actually much less confusing to just use the annotation.